test/Microsoft.DotNet.Tools.Publish.Tests/Microsoft.DotNet.Tools.Publish.Tests.cs

```diff
@@ -213,17 +213,26 @@ public void PublishScriptsRun()
             result.Should().Pass();
         }
 
+        private void GenerateTempGlobalJson(TempDirectory tempDir, string repoRoot)
```

Comment:
Yeah I think you are right here. 

I envisioned that this method would be eventually moved to another place and wanted to make that easy.
I'll change it.